src/ert/config/refcase.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import (
+    Any,
+    List,
+    Optional,
+    Sequence,
+)
+
+import numpy as np
+import numpy.typing as npt
+
+from ._read_summary import read_summary
+from .parsing.config_dict import ConfigDict
+from .parsing.config_errors import ConfigValidationError
+from .parsing.config_keywords import ConfigKeys
+
+
+@dataclass(eq=False)
+class Refcase:
```

Comment:
Why did this move?

Reply:
To avoid circular import. Was in `EnsembleConfig`, which indirectly instantiates/imports `SummaryConfig`. Moving some `Refcase` logic to `SummaryConfig` makes `SummaryConfig` have to import it, and it can't be from `EnsembleConfig`.